index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Md. Porag Sarkar: Portfolio</title>
+</head>
+
+<body>
+    <div class="container-parent">
+
+        <header class="top-navigation">
+            <div class="container-logo">
+                <p class="logo">My Logo</p>
```

Comment:
- [ ] Your logo looks really nice and is in the correct position:+1: However it will be better if we use an `a` tag, This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.